packages/app-store/alby/api/add.ts

```diff
@@ -1,4 +1,47 @@
-import { createDefaultInstallation } from "@calcom/app-store/_utils/installation";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import config from "../config.json";
+
+// FIXME: is a custom handler really needed?
```

Comment:
@hariombalhara I would leave this to you since you have more knowledge of appStore

Reply:
I updated the comment. The only thing we want to do here is redirect to the setup page after installing the app.